### PR TITLE
Load json file for topology

### PR DESCRIPTION
**Pull request by minsulee2**
Some updates:
- Enable loading json file to draw topology graph
- Add example json file

### CODE REVIEW

File: tracker-server/example.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "nodes": [
+    {
+      "address": "t"
```
Comment (platfowner) [inline tracker-server/example.json:4]: minor) what about more friendly address samples e.g. "0x001", "0xADDR_001", ...?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: tracker-server/example.json (hunk 1 above)
Comment (minsulee2) [reply tracker-server/example.json]: Updated!

**Comment by minsulee2** [comment]
Thanks for the fast review! Merging now !

**Pull request merged**